DataConnectors/Okta Single Sign-On/AzureFunctionOktaSSO_V2/AzureFunctionOktaSSO/run.ps1

```diff
@@ -144,14 +145,18 @@ do {
             -method $method `
             -contentType $contentType `
             -resource $resource
-        $LAuri = "https://" + $customerId + ".ods.opinsights.azure.com" + $resource + "?api-version=2016-04-01"
+		if ([string]::IsNullOrEmpty($logAnalyticsUri))
```

Comment:
This check is not required, as we are dynamically generating LA URI in ARM Template and send it as environment variable to Azure function. The value will not be null or empty

Reply:
The package will be shared by new installation and existing installation, we will do an in-place upgrade. This check is for backward compatibility, for existing customer with already installed azure function, the current function doesn't have this environment variable defined.

Reply:
Resolving as the change is needed.